build.md

```diff
@@ -0,0 +1,145 @@
+# Building Volt
+
+###### Note: This repository can be built with the default SDK but will not be outlined in this guide.
+###### Note: x64 platform builds will also not be outlined in this guide.
+
+All build instructions start the same. Clone a local copy of [mini-source-sdk](https://github.com/Joshua-Ashton/mini-source-sdk). (A minified version of [Source SDK](https://github.com/ValveSoftware/source-sdk-2013) with support for VS 2022)
```

Comment:
Should probably add that you need VS 2022 with the Windows 11 SDK to build.

Reply:
What do you mean specifically? When I built this I did it on Windows 10 :p

Reply:
You can still install the Windows 11 SDK. It fixes this issue.

Reply:
I just read the comment below, sounds good.